content/docs/components/serving/overview.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Overview"
+description = "Model Serving Overview"
```

Comment:
Change the description to sentence case rather than title case: "Model serving overview".

Capital letters can be confusing, as they lead people to think there's a brand name involved. For more info about capitalization, see our [style guide](https://www.kubeflow.org/docs/about/style-guide/).